ci_framework/plugins/action/ci_kustomize.py

```diff
@@ -0,0 +1,755 @@
+#!/usr/bin/env python3
+
+# Copyright Red Hat, Inc.
+# Apache License Version 2.0 (see LICENSE)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+action: ci_kustomize
+
+short_description: Applies a set of k8s Kustomizations to a set of manifests
+
+description:
+- Allows applying a set of Kustomizations to a given set of manifests using
+  the kustomize or oc tools.
+- kustomize or oc should be discoverable by in the PATH.
+- This modules takes a set of manifest files pointed by I(target_path) and
+  applies to them the set of kustomizations, if available.
+- The kustomization result is always saved in a single file and passed to
+  the caller in the O(result) field if success.
```

Comment:
but is that one typo? O(

Reply:
And this one references and `output` field

Reply:
thanks